### ISSUE TITLE
Add PHP 8 support

### ISSUE THREAD

**Issue by roy-bongers** [transip/transip-api-php#17]
Installing the API as a composer package when using PHP 8 fails. Please update the requirements in the `composer.json` file.

```
$ composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - transip/transip-api-php is locked to version v6.3.0 and an update of this package was not requested.
    - transip/transip-api-php v6.3.0 requires php ^7.2.0 -> your php version (8.0.0) does not satisfy that requirement.
```

**Comment by bedrijfsportaal** [comment]
Same issue...:
`- transip/transip-api-php v6.6.0 requires php ^7.2.0 -> your php version (8.0.2) does not satisfy that requirement.`

**Comment by samihsoylu** [comment]
This issue has been addressed in pull request #18
